lexers/t/teraterm.go

```diff
@@ -0,0 +1,56 @@
+package t
+
+import (
+	"regexp"
+
+	. "github.com/alecthomas/chroma" // nolint
+	"github.com/alecthomas/chroma/lexers/internal"
+)
+
+var teraTermAnalyserCommandRe = regexp.MustCompile(
+	`(?i)\b(basename|beep|bplusrecv|bplussend|break|bringupbox|` +
+		`callmenu|` + // 'call' is handled separately.
+		`changedir|checksum16|checksum16file|checksum32|checksum32file|checksum8|checksum8file` +
```

Comment:
This is quite hard to read. I guess it would be better to do it more or less similar to pygments code and split it into a string slice like:
```go
regexp.MustCompile(`(?i)\b(` + strings.Join([]string{
    "basename",
    "beep",
    ...
}, "|") + `)\b`)
```

Reply:
great! tks